plugins/kinet/README.md

```diff
@@ -2,12 +2,34 @@ KiNET Plugin
 ============
 
 This plugin creates a single device with multiple output ports. Each port
-represents a power supply. This plugin uses the V1 DMX-Out version of the
-KiNET protocol.
+represents a power supply. This plugin supports both the DMX Out (V1) and the
+Port Out (V2) modes of the KiNET protocol.
 
 
 ## Config file: `ola-kinet.conf`
 
-`power_supply = <ip>`  
+### Per Power Supply Settings
+
+`power_supply = <ip>`
 The IP of the power supply to send to. You can communicate with more than
-one power supply by adding multiple `power_supply =` lines
+one power supply by adding multiple `power_supply =` lines.
```

Comment:
I think this should probably be above the sub heading but in the config file bit, as it's not a setting of the PSU but something that makes them exist. Which would be more relevant if we add auto-detection.

Reply:
Moved.